pkg/uconn/analyzer.go

```diff
@@ -51,13 +56,24 @@ func (a *analyzer) close() {
 func (a *analyzer) start() {
 	a.analysisWg.Add(1)
 	go func() {
+		ssn := a.db.Session.Copy()
+		defer ssn.Close()
+
+		uconnColl := ssn.DB(a.db.GetSelectedDB()).C(a.conf.T.Structure.UniqueConnTable)
 
 		for datum := range a.analysisChannel {
 
```

Comment:
Could you add a short comment above each of the next three lines/function calls just giving a really short explanation of each?

Reply:
Added